src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java

```diff
@@ -66,6 +66,19 @@
  * <pre>
  * &lt;module name="NeedBraces"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * if (true) { // OK
+ *    return true;
+ * }  else  // violation, single line statements without braces are not allowed
+ *    return false;
+ * for(int i =0; i &lt; 5 ; i++ ) { // OK
+ *    ++count;
+ * }
+ * do // violation, single line statements without braces are not allowed
+ *    ++count;
+ *  while (false);
+ * </pre>
```

Comment:
please add violations for for- and while-cycles

Reply:
done.

Reply:
Just a suggestion, I think we should add different code examples for valid and invalid as the code would look neater, rest up to you.